### PR TITLE
Migrating keys first draft, with sidebars update.

### PR DESCRIPTION
**Pull request by PaulD-docs**
First draft of the Migrating keys documentation. This also includes the update to the sidebar to include the new Advanced use section

### CODE REVIEW

File: website/docs/advanced/migrating-keys.md [hunk 1]
```diff
@@ -0,0 +1,349 @@
+---
+id: migrating-keys
+title: Migrating Validator Keys
```
Comment (rauljordan) [inline website/docs/advanced/migrating-keys.md:3]:
```suggestion
title: Migrating validator keys between computers
```

File: website/docs/advanced/migrating-keys.md [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+sidebar_label: Migrating Validator Keys
```
Comment (rauljordan) [inline website/docs/advanced/migrating-keys.md:4]:
```suggestion
sidebar_label: Migrating between computers
```

File: website/docs/advanced/migrating-keys.md [hunk 3]
```diff
@@ -0,0 +1,349 @@
+---
+id: migrating-keys
+title: Migrating Validator Keys
+sidebar_label: Migrating Validator Keys 
+---
+
+The aim of this document is to provide guidance on migrating validator keys from one active system to another whilst minimising the risk of slashing.
```
Comment (rauljordan) [inline website/docs/advanced/migrating-keys.md:7]:
```suggestion
The aim of this document is to provide guidance on migrating Prysm validator keys from one system to another whilst minimising the risk of slashing.  
```

File: website/docs/advanced/migrating-keys.md [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+title: Migrating Validator Keys
+sidebar_label: Migrating Validator Keys 
+---
+
+The aim of this document is to provide guidance on migrating validator keys from one active system to another whilst minimising the risk of slashing.  
+
+:::danger Important Note
+As there is a risk of slashing when undertaking migration, it is important to ensure the following key points are understood and followed: 
+
+1.	Never run more than a single Validator process with the same keys loaded
```
Comment (rauljordan) [inline website/docs/advanced/migrating-keys.md:12]:
```suggestion
1.	Never run more than a single validator process with the same keys loaded
```

File: website/docs/advanced/migrating-keys.md [hunk 5]
```diff
@@ -0,0 +1,349 @@
...
+
+```sh
+bazel run //validator -- slashing-protection import --datadir=/path/to/your/wallet --slashing-protection-json-file=/path/to/desiredimportfile
+```
+
+</TabItem>
+</Tabs>
+
+
+### Step 5:  Verification and restart Validator
```
Comment (rauljordan) [inline website/docs/advanced/migrating-keys.md:314]:
```suggestion
### Step 5:  Verification and restarting the validator client
```

**Comment by PaulD-docs** [comment]
All edits agreed and approved. Have also added in the beacon node as an API page.  Not sure why this has appeared in this PR?

**Comment by PaulD-docs** [comment]
> First draft of the Migrating keys documentation. This also includes the update to the sidebar to include the new Advanced use section

All comments have been reviewed and approved where needed and this should now be ready for a final review and merging. It contains two new files, migrating keys and using the beacon as an API along with an update to the sidebars file to add the Advanced usage section.

**Comment by har00ga** [comment]
Just made a couple adjustments to phrasing throughout the two pieces. Great work, thanks for the contribution! 🙂

**Review by har00ga — APPROVED** [review]

**Pull request merged**